pywinauto/timings.py

```diff
@@ -319,11 +319,45 @@ def WaitUntil(
             
     return func_val
 
+#=========================================================================
+def wait_until(

```

Comment:
The idea is to keep the implementation in one place but still have the both
the decorator and the regular function. The first comment is only about
sorting out right names and aliases according to the convention in the
project.

On Sun, 17 Apr 2016, 22:59 Denis Anufriev, <notifications@github.com> wrote:

> In pywinauto/timings.py
> <https://github.com/pywinauto/pywinauto/pull/180#discussion_r59982141>:
>
> > @@ -319,11 +319,45 @@ def WaitUntil(
> >
> >      return func_val
> >
> > +#=========================================================================
> > +def wait_until(
>
> I'm not sure that understood this comment right. How it should collaborate
> with first comment?
>
> def WaitUntill(timeout,
>            retry_interval,
>            func,
>            value = True,
>            op = operator.eq,
>            *args):
>
> return wait_until(timeout, retry_interval, value, op)(func)(*args)
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/pywinauto/pywinauto/pull/180/files/7d46c8016e1ce4a70d63605937d5fb533e073168#r59982141>
>
-- 
Cheers,
Valentin